code/datums/components/crafting/recipes/recipes_clothing.dm

```diff
@@ -39,6 +39,30 @@
 	time = 10
 	reqs = list(/obj/item/paper = 20)
 	category = CAT_CLOTHING
+	
+/datum/crafting_recipe/armwraps
+	name = "armwraps"
+	result = /obj/item/clothing/gloves/fingerless/pugilist
+	time = 60
+	tools = list(TOOL_WIRECUTTER)
+	reqs = list(/obj/item/stack/sheet/cloth = 4,
+				/obj/item/stack/sheet/silk = 2,
+				/obj/item/stack/sheet/leather = 2)
+	category = CAT_CLOTHING
+
+/datum/crafting_recipe/armwrapsplusone
+	name = "armwraps of mighty fists"
+	result = /obj/item/clothing/gloves/fingerless/pugilist/magic
+	time = 300
+	tools = list(TOOL_WIRECUTTER, /obj/item/book/codex_gigas, /obj/item/clothing/head/wizard, /obj/item/clothing/suit/wizrobe)
+	reqs = list(/obj/item/stack/sheet/cloth = 2,
+				/obj/item/stack/sheet/leather = 2,
+				/obj/item/stack/sheet/durathread = 2,
+				/datum/reagent/consumable/ethanol/sake = 100,
+				/datum/reagent/consumable/ethanol/wizz_fizz = 100,
+				/obj/item/stack/sheet/sinew = 1,
```

Comment:
maybe not sinew and 50 sheets of gold
Silk and 10 sheets of gold with 5 sheets of silver would be better?

Reply:
50 gold sheets is 1000 units of gold reagent. Thats the best approximation to 1000 gold pieces I could use for an elaborate 3.
X dnd joke without making someone quite literally go and press 1000 gold coins from the horrifically archaic coin press. 1000 gold pieces was the price of a +1 magic weapon you make yourself. Technically this is a flawed joke as amulets of mighty fists, which is what this item is a reference too, is 1500 gold pieces due to dumb pricing shit, but you get the idea. Nobody is going to call me out on an extremely obtuse joke.

Also the sinew isn't that hard to acquire. The idea is that it uses rare and interesting materials as components. Sinew means you need to either hit up xenobio or a miner, wizz fizz and sake the bartender, durathread the botanists (as well as the cloth and leather probably), and the gold from hopefully mining doing their job. Magical items were always the subject of elaborate bullshit and this gives some reasonable deniability to whether it's magic or the materials used to make the item making it distinctly durable and the enhancement a placebo. You just THINK it should hit harder, so you hold back a little less.

Thank you for coming to my TEDTalk where I explain 'the joke that nobody asked for'.